pkg/analysis/references/testdata/src/a/a.go

```diff
@@ -0,0 +1,22 @@
+package a
+
+type TestSpec struct {
```

Comment:
Can you try some field names where they contain the string `ref`/`refs` or `reference`/`references` but not at the end? Eg `Prefix` or `Prefereneces` perhaps could be in field names?